### DISCUSSION TITLE
Refresh Rate Implementation in Lowdefy (Refreshing API, not page refresh)

### DISCUSSION THREAD

**Discussion by haseeb-xd** [lowdefy/lowdefy#1026] [Questions and Help]
Hello Everyone,

I want to implement refresh functionality in Lowdefy. I want to hit the specific API every 10 seconds and update that data in the Echart provided by Lowdefy. 

Hoping to get an answer .

**Comment by Gervwyk** [comment]
Hi @haseeb-xd 

There is a working example of the in this discussion, you just need to reference the request-response of the data to the chart https://github.com/lowdefy/lowdefy/discussions/878#discussioncomment-1402342

In future versions we plan to have an official version of this functionality that does not require a custom JsAction.
